pkgs/native_assets_cli/example/build/linked_library/README.md

```diff
@@ -0,0 +1,7 @@
+An example library that builds two native libraries where one library
```

Comment:
`linked_library` could mean many things. Let's try to find a more specific name: It has to do with a native lib having a dependency on another native lib, and using it via the C linker and then loading it at runtime via de C runtime. 

So it's linked yes, but it's dynamically linked (not statically. That would be trivial from a bundling perspective.)

And the dependency is being referred from from native code, not Dart code.

So a good name for this example directory would be `native_dynamic_linking`?